dietpi/dietpi-software

```diff
@@ -8125,7 +8132,26 @@ _EOF_
 			G_EXEC_NOHALT=1 G_EXEC rm install.bash
 
 		fi
+		
+                software_id=203 # Zerotier
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+                        Banner_Installing
+			 
+			#Purge down in case of reinstall
+                        [[ -f '/usr/sbin/zerotier-cli' ]] && G_EXEC rm /usr/sbin/zerotier-cli
+                        [[ -f '/usr/sbin/zerotier-one' ]] && G_EXEC rm /usr/sbin/zerotier-one
+                        [[ -f '/etc/apt/sources.list.d/zerotier.list' ]] && G_EXEC rm /etc/apt/sources.list.d/zerotier.list
+                        [[ -f '/var/lib/zerotier-one/identity.public' ]] && G_EXEC rm /var/lib/zerotier-one/identity.public
```

Comment:
This might not be needed if we setup the source list and public key directly. Usually this will not change and could stay on a reinstall.